src/test/java/org/prebid/server/validation/BidderParamValidatorTest.java

```diff
@@ -335,7 +338,7 @@ public void validateShouldReturnValidationMessagesWhenEplanningExtNotValid() {
     @Test
     public void validateShouldNotReturnValidationMessagesWhenSomoaudienceImpExtIsOk() {
         // given
-        final ExtImpSomoaudience ext = ExtImpSomoaudience.of("placementId");
+        final ExtImpSomoaudience ext = ExtImpSomoaudience.of("placementId", BigDecimal.valueOf(1.1111));
```

Comment:
Prices usually have 2 digits after decimal point, let's follow the same rule here